### PR TITLE
Re #21 - Add matchall filter type

### PR DESCRIPTION
**Pull request by sophacles**
This tc filter matches all packets that encounter it.

### CODE REVIEW

**Comment by wllenyj** [comment]
Thanks, 
Please fix the ci fail.

**Comment by sophacles** [comment]
Hi, I've rebased and fixed the formatting errors, will pay attention to the next CI run for additonal issues.

**Review by wllenyj — APPROVED** [review]

**Comment by wllenyj** [comment]
@cathay4t Do you have any comments?

File: src/rtnl/tc/constants.rs [hunk 1]
```diff
@@ -20,6 +20,15 @@ pub const TC_H_MIN_PRIORITY: u32 = 0xFFE0;
...
 pub const TC_H_MIN_EGRESS: u32 = 0xFFF3;
 
+// matchall filters
+pub const TCA_MATCHALL_UNSPEC: u16 = 0;
+pub const TCA_MATCHALL_CLASSID: u16 = 1;
+pub const TCA_MATCHALL_ACT: u16 = 2;
+pub const TCA_MATCHALL_FLAGS: u16 = 3;
+pub const TCA_MATCHALL_PCNT: u16 = 4;
+pub const TCA_MATCHALL_PAD: u16 = 5;
+pub const TCA_MATCHALL_MAX: u16 = TCA_MATCHALL_PAD;
```
Comment (cathay4t) [inline src/rtnl/tc/constants.rs:30]: Please remove `TCA_MATCHALL_PAD` and `TCA_MATCHALL_MAX`. They are not used in this PR

**Review by cathay4t — CHANGES_REQUESTED** [review]

**Comment by cathay4t** [comment]
@sophacles Thanks for the patch. I have amended the patch to hold `Signed-off-by:` in git commit message. Next time, please use `git commit --signoff`.

**Pull request merged**